firestore/client.go

```diff
@@ -290,10 +290,20 @@ func (c *Client) Collections(ctx context.Context) *CollectionIterator {
 }
 
 // Batch returns a WriteBatch.
+//
+// Deprecated: The WriteBatch API has been replaced with the transaction and
+// the bulk writer API. For atomic transaction operations, use `Transaction`.
+// For bulk read and write operations, use `BulkWriter`.
 func (c *Client) Batch() *WriteBatch {
 	return &WriteBatch{c: c}
 }
 
+// BulkWriter returns a BulkWriter instance.
```

Comment:
Any time a context is going to be stored and held and not acted upon immediately it is best practice to outline this in the docs and how it will be used.

Reply:
Done.